pennylane/operation.py

```diff
@@ -310,25 +310,25 @@ def matrix(self):
     def name(self, value):
         self._name = value
 
-    def __init__(self, *params, wires=None, do_queue=True):
+    def __init__(self, *data, wires=None, do_queue=True):
```

Comment:
Same comment here, this should remain 'params' since its the argument to the class.